### ISSUE TITLE
The Future of navnløs / developement plans

### ISSUE THREAD

**Issue by joseywoermann** [joseywoermann/navnlos#176]
## TLDR: I have decided to stop development on navnløs indefinitely.

It will continue to receive security patches and bug fixes, but there will not be any new features or major changes.
Additionally, I will remove all public invite links, which means that the bot will not be able to join any more servers.

However, the bot will remain on all servers it currently is on and will continue running like normal. I don't plan on shutting it down any time soon.
This step shouldn't come as a surprise to those who saw the early days of the bot. It was never meant to be a public bot and was originally only designed for a single server.
